src/assets/styles/components/_nav.scss

```diff
@@ -114,11 +114,13 @@
 .menu a,
 .menu button {
   align-items: center;
+  border-radius: 5px;
```

Comment:
I don't believe there should be a border radius on these.

Reply:
oh, we don't want rounded outline on focus/hover?

Reply:
No, the primary navigation doesn't have that effect, see https://idrc.ocadu.ca. The rounding is only for links within the content and footer.

Reply:
got it, I will remove it 👍